libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp

```diff
@@ -153,7 +153,7 @@ void GLBackend::init() {
 
         if (vendor.contains("NVIDIA") ) {
             qCDebug(gpugllogging) << "NVIDIA card detected";
-#if !defined(Q_OS_ANDROID)
+#if !defined(Q_OS_ANDROID) && !defined(USE_GLES)
```

Comment:
I looked at the code a bit to refresh my memory. I think it'd be better to:

1. modify availableMemoryKnown() to return false. That'll disable the "use free VRAM" system, and default to a fraction of total VRAM.
2. Check if GLBackend::getTotalMemory() returns something sensible. If it does, it's fine. If it doesn't, either fix it or ensure it returns 0, in which case we revert to the old system of a hardcoded 2GB VRAM limit.

Reply:
Problem is that it doesn't build with any of that stuff. Not sure if it is because of GLES or because of the other `glad` package.
```
/home/motofckr9k/Vircadia/source/libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp: In lambda function:
/home/motofckr9k/Vircadia/source/libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp:109:44: error: ‘GL_GPU_MEMORY_INFO_DEDICATED_VIDMEM_NVX’ was not declared in this scope
 #define GL_GET_INTEGER(NAME) glGetIntegerv(GL_##NAME, &const_cast<GLint&>(NAME));
                                            ^
/home/motofckr9k/Vircadia/source/libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp:109:44: note: in definition of macro ‘GL_GET_INTEGER’
 #define GL_GET_INTEGER(NAME) glGetIntegerv(GL_##NAME, &const_cast<GLint&>(NAME));
                                            ^~~
/home/motofckr9k/Vircadia/source/libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp:109:44: note: suggested alternative: ‘GPU_MEMORY_INFO_DEDICATED_VIDMEM_NVX’
 #define GL_GET_INTEGER(NAME) glGetIntegerv(GL_##NAME, &const_cast<GLint&>(NAME));
                                            ^
/home/motofckr9k/Vircadia/source/libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp:109:44: note: in definition of macro ‘GL_GET_INTEGER’
 #define GL_GET_INTEGER(NAME) glGetIntegerv(GL_##NAME, &const_cast<GLint&>(NAME));
                                            ^~~
/home/motofckr9k/Vircadia/source/libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp:109:44: error: ‘GL_GPU_MEMORY_INFO_TOTAL_AVAILABLE_MEMORY_NVX’ was not declared in this scope
 #define GL_GET_INTEGER(NAME) glGetIntegerv(GL_##NAME, &const_cast<GLint&>(NAME));
                                            ^
/home/motofckr9k/Vircadia/source/libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp:109:44: note: in definition of macro ‘GL_GET_INTEGER’
 #define GL_GET_INTEGER(NAME) glGetIntegerv(GL_##NAME, &const_cast<GLint&>(NAME));
                                            ^~~
/home/motofckr9k/Vircadia/source/libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp:109:44: note: suggested alternative: ‘GPU_MEMORY_INFO_TOTAL_AVAILABLE_MEMORY_NVX’
 #define GL_GET_INTEGER(NAME) glGetIntegerv(GL_##NAME, &const_cast<GLint&>(NAME));
                                            ^
/home/motofckr9k/Vircadia/source/libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp:109:44: note: in definition of macro ‘GL_GET_INTEGER’
 #define GL_GET_INTEGER(NAME) glGetIntegerv(GL_##NAME, &const_cast<GLint&>(NAME));
                                            ^~~
/home/motofckr9k/Vircadia/source/libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp:109:44: error: ‘GL_GPU_MEMORY_INFO_CURRENT_AVAILABLE_VIDMEM_NVX’ was not declared in this scope
 #define GL_GET_INTEGER(NAME) glGetIntegerv(GL_##NAME, &const_cast<GLint&>(NAME));
                                            ^
/home/motofckr9k/Vircadia/source/libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp:109:44: note: in definition of macro ‘GL_GET_INTEGER’
 #define GL_GET_INTEGER(NAME) glGetIntegerv(GL_##NAME, &const_cast<GLint&>(NAME));
                                            ^~~
/home/motofckr9k/Vircadia/source/libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp:109:44: note: suggested alternative: ‘GPU_MEMORY_INFO_CURRENT_AVAILABLE_VIDMEM_NVX’
 #define GL_GET_INTEGER(NAME) glGetIntegerv(GL_##NAME, &const_cast<GLint&>(NAME));
                                            ^
/home/motofckr9k/Vircadia/source/libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp:109:44: note: in definition of macro ‘GL_GET_INTEGER’
 #define GL_GET_INTEGER(NAME) glGetIntegerv(GL_##NAME, &const_cast<GLint&>(NAME));
                                            ^~~
/home/motofckr9k/Vircadia/source/libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp:109:44: error: ‘GL_TEXTURE_FREE_MEMORY_ATI’ was not declared in this scope
 #define GL_GET_INTEGER(NAME) glGetIntegerv(GL_##NAME, &const_cast<GLint&>(NAME));
                                            ^
/home/motofckr9k/Vircadia/source/libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp:109:44: note: in definition of macro ‘GL_GET_INTEGER’
 #define GL_GET_INTEGER(NAME) glGetIntegerv(GL_##NAME, &const_cast<GLint&>(NAME));
                                            ^~~
/home/motofckr9k/Vircadia/source/libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp:109:44: note: suggested alternative: ‘TEXTURE_FREE_MEMORY_ATI’
 #define GL_GET_INTEGER(NAME) glGetIntegerv(GL_##NAME, &const_cast<GLint&>(NAME));
                                            ^
/home/motofckr9k/Vircadia/source/libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp:109:44: note: in definition of macro ‘GL_GET_INTEGER’
 #define GL_GET_INTEGER(NAME) glGetIntegerv(GL_##NAME, &const_cast<GLint&>(NAME));
                                            ^~~
/home/motofckr9k/Vircadia/source/libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp: In static member function ‘static size_t gpu::gl::GLBackend::getAvailableMemory()’:
/home/motofckr9k/Vircadia/source/libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp:229:27: error: ‘GL_GPU_MEMORY_INFO_CURRENT_AVAILABLE_VIDMEM_NVX’ was not declared in this scope
             glGetIntegerv(GL_GPU_MEMORY_INFO_CURRENT_AVAILABLE_VIDMEM_NVX, &mem[0]);
                           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/motofckr9k/Vircadia/source/libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp:229:27: note: suggested alternative: ‘GPU_MEMORY_INFO_CURRENT_AVAILABLE_VIDMEM_NVX’
             glGetIntegerv(GL_GPU_MEMORY_INFO_CURRENT_AVAILABLE_VIDMEM_NVX, &mem[0]);
                           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                           GPU_MEMORY_INFO_CURRENT_AVAILABLE_VIDMEM_NVX
/home/motofckr9k/Vircadia/source/libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp:234:27: error: ‘GL_TEXTURE_FREE_MEMORY_ATI’ was not declared in this scope
             glGetIntegerv(GL_TEXTURE_FREE_MEMORY_ATI, &mem[0]);
                           ^~~~~~~~~~~~~~~~~~~~~~~~~~
/home/motofckr9k/Vircadia/source/libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp:234:27: note: suggested alternative: ‘TEXTURE_FREE_MEMORY_ATI’
             glGetIntegerv(GL_TEXTURE_FREE_MEMORY_ATI, &mem[0]);
                           ^~~~~~~~~~~~~~~~~~~~~~~~~~
                           TEXTURE_FREE_MEMORY_ATI
```